build.yaml

```diff
@@ -1,14 +1,17 @@
 ---
 # We just wrap `build` so this is really it
 name: "jellyfin-ffmpeg"
-version: "4.2.1-2"
+version: "4.2.1-4"
```

Comment:
curious - where did 3 go?

Reply:
It existed, but we never updated the build.yaml - doesn't seem to have mattered though.